### PR TITLE
Update Readme to include branching strategy

### PR DESCRIPTION
**Pull request by ggjjj**
Readme in dunfell branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,23 @@ installation and configuration.
 [packaging]: https://github.com/Azure/iotedge/blob/main/doc/packaging.md
 [updating-guide]: https://learn.microsoft.com/azure/iot-edge/how-to-update-iot-edge?view=iotedge-1.4&tabs=ubuntu#special-case-update-from-10-or-11-to-latest-release
 
+
+Branching Strategy and Timelines
+===============================
+
+| Yoto | IoT Edge version | Branch Name | Branch Status |
```
Comment (micahl) [inline README.md:76]:
```suggestion
| Yocto release | IoT Edge version | Branch Name | Branch Status |
```

File: README.md [hunk 2]
```diff
@@ -69,6 +69,23 @@ installation and configuration.
...
+| :- | :- | :- | :- |
+| Dunfell | 1.4.x  | main | Active and maintained |
+| Dunfell | 1.4.x  | dunfell | Active and maintained |
+| Dunfell | 1.1.x  | dunfell-1.1 | 1.1.x is deprecated in Dec'22. Not maintained |
+| Kirkstone | 1.1.x | N/A | 1.1.x is deprecated in Dec'22 |
+| Kirkstone | 1.4.x | N/A | ETA TBD |
```
Comment (micahl) [inline README.md:82]: I think we can drop the Kirkstone + 1.1 entry and just move the Kirkstone + 1.4 entry to be the first one.  I'm assuming that if (when) we add it the codeflow would be to first update the main (master) branch and then fork a Kirkstone branch.

**Review by micahl — CHANGES_REQUESTED** [review]

**Comment by ggjjj** [comment]
Do we need to do the same for master branch ? @micahl

**Comment by ggjjj** [comment]
> > Do we need to do the same for master branch ? @micahl
> 
> Ah, right. I assumed this was for the master branch. :) What if we only do this for the master branch? Duplicating this same info and keeping it up to date across multiple branches seems like it would add extra overhead.
> 
> Btw, one more minor typo is to capitalize 'thud' in the Yocto release column.

sounds good. I will close this and create a new PR as it would have so many conflicts

**Pull request closed without merging**